Waidhoferj/CSC-308-309-Project
# Braden50/report content page

Added the report page on the frontend and its functionality in the back

There were also a couple of edits I made to fix up other mutations (ex: Group and artwork)
Adding testing to make sure we aren't missing anything when a report is submitted

NOTE: Frontend feels weird without a "you submitted a report!" kind of response. I wasn't sure how to do this exactly in due time. 

To test:

1. Boot up the frontend and backend ("python app.py -l")
2. Go to the map
3. Look at an already discovered artwork
4. "View in my portfolio"
5. Click the report button
6. Fill out and submit the form
7. Run the query below on the backend (http://localhost:5000/graphql) to make sure your report is now in the database
     - NOTE: You can use reports(last: 1) { ... } to see the most recent report.

{ 
  reports {
    edges {
      node {
        reportedIdType
        reportedId
        userId
        reason
        description
        dateSubmitted
      }
    }
  }
}
